datalad/utils.py

```diff
@@ -91,16 +96,12 @@ def get_linux_distribution():
     return result
 
 
-try:
-    linux_distribution_name, linux_distribution_release \
-        = get_linux_distribution()[:2]
-    on_debian_wheezy = on_linux \
-                       and linux_distribution_name == 'debian' \
-                       and linux_distribution_release.startswith('7.')
-except:  # pragma: no cover
-    # MIH: IndexError?
-    on_debian_wheezy = False
-    linux_distribution_name = linux_distribution_release = None
+# TODO: deprecated, remove in 0.15
+# Wheezy EOLed 2 years ago, no new datalad builds from neurodebian
+on_debian_wheezy = False
```

Comment:
Hm. I don't know where this is actually used. `git grep` says nowhere, but I have no clue about building debian package. However, if it is used anywhere, it's a bit weird to claim `on_debian_wheezy` is `False`. Either the name is suggesting the wrong thing ("Are we on wheezy?") or it should not report a non-factual thing. And if it's not used anywhere in the build process, then just remove it entirely?

Reply:
It is "factual" (with 99.9% confidence) since it **very** unlikely anyone would run it on EOLed wheezy. We aren't even compatible with its python3 if it had any(next one, oldoldstable jessie had 3.4).
I didn't want to just remove it since  if some (elderly) code in some extension or 3rd party code DOES use it for some reason, it would fail -- not something we would like to cause by updates in `maint` branch.  We will remove it in some next release though (as comment says).

Reply:
True. I didn't notice it's against `maint`.
I'm fine with deprecating and later remove, of course. Although w/o any deprecation warning I'd be surprised if anyone actually using it notices the deprecation. So, in my estimation the removal will be as surprising as it would be in the next minor release. Anyways - don't consider this a blocker.

Reply:
In my view this variable, as well as `linux_distribution_name` and `linux_distribution_release`, are obscure and minor enough that just a mention in the 0.14.0 changelog would be fine.  

Anyway, I think we all agree that the specifics of how we handle this is unlikely to matter, and delaying/caching `get_linux_distribution` is valuable.  Will merge.